providers/universal-provider/README.md

```diff
@@ -0,0 +1,96 @@
+# @walletconnect/universal-provider
+
+Universal Provider for WalletConnect Protocol
+
+## Usage
+
+```typescript
+import { ethers } from "ethers";
+import UniversalProvider from "@walletconnect/universal-provider";
+
+//  Initialize the provider
+const provider = await UniversalProvider.init({
```

Comment:
nit: why provide the rpc in `connect` not here?

Reply:
the idea for the rpc config being in the `connect` fx and not in the `init` was that in some use-cases you might not know which chains will be used at the time of initialization